src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/configuration/KafkaConsumerConfig.java

```diff
@@ -63,7 +68,8 @@
      */
     private ConsumerFactory<String, ChipsRestInterfacesSend> newMainConsumerFactory() {
```

Comment:
we should probably ensure that the error handling deserializer is used for the main, retry and error consumers